src/MIDebugPackage/OpenFolderSchema.json

```diff
@@ -207,18 +219,35 @@
                     "description": "If individual arguments contain characters (such as spaces or tabs), should it be quoted? Defaults to 'true'. If set to false, the debugger command will no longer be automatically quoted."
                   }
                 }
+              },
+              "sourceFileMapOptions": {
+                "type": "object",
+                "properties": {
+                  "editorPath": {
+                    "type": "string",
+                    "description": "The location of the source code for the editor to locate."
+                  },
+                  "useForBreakpoints": {
+                    "type": "boolean",
+                    "description": "Use this source mapping for breakpoint binding? Default is true."
```

Comment:
>for breakpoint binding [](start = 60, length = 22)

I wonder if people will understand what 'for breakpoint binding' means?

How about:
Use this source mapping when setting breakpoints? Default is 'true'. Set to 'false' if files under 'editorPath' were not always originally compiled from the compiler path. Note that if if a source file doesn't match any entry in the map, the breakpoint will be set using just the file name.